### PR TITLE
Added new development wallpapers for ubuntu-core-desktop

### PR DESCRIPTION
**Pull request by kenvandine**

### CODE REVIEW

File: hooks/006-add-gdm.chroot [hunk 1]
```diff
@@ -132,6 +132,14 @@ sed -i "/^ReadWritePaths=/ a \\  /var/lib/extrausers/ \\\\" \
 # Hide gnome-terminal by default
 sed -i 's/OnlyShowIn=/NoDisplay=true\nOnlyShowIn=/g' /usr/share/applications/org.gnome.Terminal.desktop
 
+# change default wallpapers
+cp /usr/share/gnome-background-properties/ubuntu-wallpapers.xml /usr/share/gnome-background-properties/ubuntu-core-desktop-wallpapers.xml
+sed -i 's/warty-final-ubuntu.png/ubuntu-core-desktop-l.png/g' /usr/share/gnome-background-properties/ubuntu-core-desktop-wallpapers.xml
```
Comment (sergio-costas) [inline hooks/006-add-gdm.chroot:137]: these two files seem to don't exist in core-base-desktop...

**Review by sergio-costas — COMMENTED** [review]
Were are those "ubuntu-core-..." background files?

File: hooks/006-add-gdm.chroot (hunk 1 above)
Comment (kenvandine) [reply hooks/006-add-gdm.chroot:137]: They are added under the static directory as part of this PR

Comment (sergio-costas) [reply hooks/006-add-gdm.chroot:137]: Oh, it's true!

**Review by sergio-costas — APPROVED** [review]
LGTM

**Pull request merged**